### ISSUE TITLE
False negative in selector-max-universal

### ISSUE THREAD

**Issue by vankop** [stylelint/stylelint#4233]
> Clearly describe the bug

`selector-max-universal` produce two warnings for `:not(*)`

> Which rule, if any, is the bug related to?

`selector-max-universal`

> What CSS is needed to reproduce the bug?

e.g.

```css
:not(*) {}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "rules": {
    "selector-max-universal": 0
  }
}
```

> Which version of stylelint are you using?

latest

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

stylelint REPL

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

no

> What did you expect to happen?

1 warning

> What actually happened (e.g. what warnings or errors did you get)?

2 warnings with same `line:column`

**Comment by evilebottnawi** [comment]
@vankop let's keep one issue for this problem instead create multiple issues
